debug/items/debug-demo-inventory.json5

```diff
@@ -0,0 +1,9 @@
+{
+    catalogue: "debug-game-items", //catalogue array or string filename of json5 catalogue
+    include: ["starter-item"], //include all items with this tag in this collection
+    defaultQty: 0, //default quantity unless specified
+    defaults: [
+        { id: "coin", qty: 1000 },
```

Comment:
o i c, not a bug but a feature? It looks odd tho since the 1000 isn't applied at construct time for some reason.

Reply:
hmm dunno, is it cause the defaultQty is 0? might be a bug made visible by this 👍

Reply:
oh if its local storage then fine :D